src/index.js

```diff
@@ -0,0 +1,945 @@
+    },
+  },
+  {
+    class: ['ArrowUp', 'special'],
+    eng: {
+      shiftUp: '▲',
+      shiftDown: '▲',
+    },
+    ru: {
+      shiftUp: '▲',
+      shiftDown: '▲',
+    },
+  },
+  {
+    class: ['ShiftRight', 'special'],
+    eng: {
+      shiftUp: 'Shift',
+      shiftDown: 'Shift',
+    },
+    ru: {
+      shiftUp: 'Shift',
+      shiftDown: 'Shift',
+    },
+  },
+];
+
+const keyRow5 = [
+  {
+    class: ['ControlLeft', 'special'],
+    eng: {
+      shiftUp: 'Ctrl',
+      shiftDown: 'Ctrl',
+    },
+    ru: {
+      shiftUp: 'Ctrl',
+      shiftDown: 'Ctrl',
+    },
+  },
+  {
+    class: ['MetaLeft', 'special'],
+    eng: {
+      shiftUp: 'Win',
+      shiftDown: 'Win',
+    },
+    ru: {
+      shiftUp: 'Win',
+      shiftDown: 'Win',
+    },
+  },
+  {
+    class: ['AltLeft', 'special'],
+    eng: {
+      shiftUp: 'Alt',
+      shiftDown: 'Alt',
+    },
+    ru: {
+      shiftUp: 'Alt',
+      shiftDown: 'Alt',
+    },
+  },
+  {
+    class: ['Space', 'special'],
+    eng: {
+      shiftUp: ' ',
+      shiftDown: ' ',
+    },
+    ru: {
+      shiftUp: ' ',
+      shiftDown: ' ',
+    },
+  },
+  {
+    class: ['AltRight', 'special'],
+    eng: {
+      shiftUp: 'Alt',
+      shiftDown: 'Alt',
+    },
+    ru: {
+      shiftUp: 'Alt',
+      shiftDown: 'Alt',
+    },
+  },
+  {
+    class: ['ArrowLeft', 'special'],
+    eng: {
+      shiftUp: '◄',
+      shiftDown: '◄',
+    },
+    ru: {
+      shiftUp: '◄',
+      shiftDown: '◄',
+    },
+  },
+  {
+    class: ['ArrowDown', 'special'],
+    eng: {
+      shiftUp: '▼',
+      shiftDown: '▼',
+    },
+    ru: {
+      shiftUp: '▼',
+      shiftDown: '▼',
+    },
+  },
+  {
+    class: ['ArrowRight', 'special'],
+    eng: {
+      shiftUp: '►',
+      shiftDown: '►',
+    },
+    ru: {
+      shiftUp: '►',
+      shiftDown: '►',
+    },
+  },
+  {
+    class: ['ControlRight', 'special'],
+    eng: {
+      shiftUp: 'Ctrl',
+      shiftDown: 'Ctrl',
+    },
+    ru: {
+      shiftUp: 'Ctrl',
+      shiftDown: 'Ctrl',
+    },
+  },
+];
+
+/* global document, localStorage */
+
+const rows = [keyRow1, keyRow2, keyRow3, keyRow4, keyRow5];
+
+const myBody = document.querySelector('body');
+
+// create wrapper for keyboard
+myBody.innerHTML =
+  '<section class="wrapper"><textarea id="text" placeholder="Write smth!" rows="15"></textarea><section class="keyboard"></section></section>';
+
+const createRow = keyRow => {
+  const row = document.createElement('div');
+  row.classList.add('row');
+
+  for (let i = 0; i < keyRow.length; i += 1) {
+    const myKey = document.createElement('div');
+    myKey.className = 'key';
+    myKey.classList.add(...keyRow[i].class);
+
+    myKey.innerHTML = `<span class="ru hide"><span class="shift up hide">${keyRow[i].ru.shiftUp}</span><span class="shift down hide">${keyRow[i].ru.shiftDown}</span></span><span class="eng"><span class="shift up">${keyRow[i].eng.shiftUp}</span><span class="shift down hide">${keyRow[i].eng.shiftDown}</span></span>`;
```

Comment:
one line code is evil. in case if you are going to insert directly HTML code, please make formatting as well.

```
 myKey.innerHTML = `
  <span class="ru hide">
    <span class="shift up hide">
       ${keyRow[i].ru.shiftUp}
    </span>
    <span class="shift down hide">
       ${keyRow[i].ru.shiftDown}
     </span>
  </span>
  <span class="eng">
    <span class="shift up">
       ${keyRow[i].eng.shiftUp}
    </span>
    <span class="shift down hide">
       ${keyRow[i].eng.shiftDown}
    </span>
  </span>
`;

```

Reply:
I tried to rewrite this, but the letters in the text area began to shift:
![image](https://user-images.githubusercontent.com/52204035/69099970-2663a900-0a6d-11ea-84bc-39e0ff0d0559.png)